content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md

```diff
@@ -0,0 +1,147 @@
+
+### **App Store open to community partners**
+
+The App Store and application lifecycle management are unique features of KubeSphere, which are based on self-developed and open-source [OpenPitrix](https://github.com/openpitrix/openpitrix).
+
+KubeSphere 3.2.0 adds the feature of **dynamically loading community-developed Helm charts into the KubeSphere App Store.** You can send a pull request containing the Helm chart of a new app to the App Store chart repository. After the pull request is merged, the app is automatically loaded to the App Store regardless of the KubeSphere version. Welcome to submit your Helm charts to https://github.com/kubesphere/helm-charts. Nocalhost and Chaos Mesh have integrated their Helms charts into KubeSphere 3.2.0 by using this method, and you can easily install them to your Kubernetes clusters by one click.
+
+![12-app-store](/images/blogs/en/release-announcement3.2.0/12-app-store.png)
+
+### **More independent KubeSphere DevOps**
+
+KubeSphere DevOps has developed into an independent project ks-devops in KubeSphere v3.2.0, which is intended to allow users to run KubeSphere DevOps in any Kubernetes clusters. Currently, you can use a Helm chart to install the backend components of ks-devops.
+
+Jenkins is a CI engine with a large user base and a rich plug-in ecosystem. In KubeSphere 3.2.0, we will let Jenkins do what it is good at—functioning only as an engine in the backend to provide stable pipeline management capability. A newly added CRD PipelineRun encapsulates run records of pipelines, which reduces the number of APIs required for directly interacting with Jenkins and boosts performance of CI pipelines.
+
+Starting from KubeSphere v3.2.0, KubeSphere DevOps allows you to build images by using pipelines based on containerd. As an independent project, KubeSphere DevOps will support independent deployment of the backend and frontend, introduce GitOps tools such as Tekton and ArgoCD, as well as integrate project management and test management platforms.
+
+### **Flexible cluster deployment**
+
+If you do not have a Kubernetes cluster, you can use KubeKey to install both Kubernetes and KubeSphere; if you already have a Kubernetes cluster, you can use ks-installer to install KubeSphere only.
+
+[KubeKey](https://github.com/kubesphere/kubekey) is an efficient open-source cluster deployment tool in the KubeSphere community, which uses Docker as the default container runtime. It can also use CRI runtimes such as containerd, CRI-O, and iSula. You can use KubeKey to deploy an etcd cluster independent of Kubernetes for better flexibility.
```

Comment:
[KubeKey](https://github.com/kubesphere/kubekey) is an efficient open-source installer.